plugins/inputs/iptables/iptables.go

```diff
@@ -32,8 +33,11 @@ func (ipt *Iptables) SampleConfig() string {
   ## iptables require root access on most systems.
   ## Setting 'use_sudo' to true will make use of sudo to run iptables.
   ## Users must configure sudo to allow telegraf user to run iptables with no password.
-  ## iptables can be restricted to only list command  "iptables -nvL"
+  ## iptables can be restricted to only list command  "iptables -nvL" or "iptables -wnvl" if using 'use_lock = true'
   use_sudo = false
+  ## Setting 'use_lock' to true will run iptables with xtables lock support.
+  ## This option is useful to avoid iptables concurrency errors when running multiple instances of this plugin.
```

Comment:
limit line length to 80 chars and mention that `use_lock` simply adds the `-w` option to the `iptables`